src/server.cpp

```diff
@@ -1919,6 +1919,16 @@ void Server::SendSetLighting(session_t peer_id, const Lighting &lighting)
 	pkt << lighting.volumetric_light_strength << lighting.shadow_tint;
 	pkt << lighting.bloom_intensity << lighting.bloom_strength_factor <<
 			lighting.bloom_radius;
+	pkt << lighting.artificial_light_color;
+	pkt << lighting.volumetric_beta_r0;
+	pkt << lighting.vignette.dark
+		<< lighting.vignette.bright
```

Comment:
Again, don't care... Might be better to follow one formatting for streaming the data